package.json

```diff
@@ -28,13 +28,27 @@
     "build-storybook": "build-storybook",
     "build": "microbundle-crl --no-compress --format modern,cjs",
     "chromatic": "npx chromatic --project-token 60pcjug3hz",
-    "lint": "eslint --ext ts,tsx src",
+    "lint123": "eslint --ext ts,tsx src",
+    "lint": "eslint --fix",
     "prepare": "npm run build",
     "start": "microbundle-crl watch --no-compress --format modern,cjs",
     "storybook": "start-storybook -p 9009",
     "test": "jest",
     "types": "tsc --noEmit"
   },
+  "husky": {
```

Comment:
Similar to the first comment, I don't think this is required anymore, as the current version of lint-staged doesn't use husky.